SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialStatics.h

```diff
@@ -22,14 +22,14 @@ struct FLockingToken
 {
 	GENERATED_USTRUCT_BODY()
 
-    UPROPERTY(BlueprintReadOnly)
+	UPROPERTY(BlueprintReadOnly, Category = "SpatialGDK|Locking")
```

Comment:
Drive-by compilation fix. Not sure under what circumstances it happens, but Engine vs Project plugin, maybe?